src/components/Mutate.js

```diff
@@ -0,0 +1,27 @@
+import { commitMutation } from 'react-relay'
```

Comment:
I think this file is an accident? It's a duplicate of the `utils/mutate` and I think you'd have a problem returning a Promise inside of a React Component :P